### PR TITLE
releases: add v9.0.0

### PR DESCRIPTION
**Pull request by benma**

### CODE REVIEW

File: releases/firmware-btc-v9.0.0/assertion.txt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+By signing this file, the signer confirms that the firmware binary built from:
+
+git tag firmware-btc-only/v9.0.0
+git commit hash 9729368ae60e52894ae3ab2be3a4ae4c47f0e03d
```
Comment (x1ddos) [inline releases/firmware-btc-v9.0.0/assertion.txt:4]: image
:+1:

File: releases/firmware-btc-v9.0.0/assertion.txt [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+
+resulted, at the time of signing, in a firmware binary file with the following sha256sum:
+
+0a8eb97828fe1300d6f85b48cb4bf5f56c2dcdb5b0e8e7289217d0aa36dfde78
```
Comment (x1ddos) [inline releases/firmware-btc-v9.0.0/assertion.txt:8]: Matches with mine. :+1:

File: releases/firmware-v9.0.0/assertion.txt [hunk 3]
```diff
@@ -0,0 +1,8 @@
+By signing this file, the signer confirms that the firmware binary built from:
+
+git tag firmware/v9.0.0
+git commit hash 9729368ae60e52894ae3ab2be3a4ae4c47f0e03d
+
+resulted, at the time of signing, in a firmware binary file with the following sha256sum:
+
+19f692a769b22abe889849d7c2987688c57489a3437e70f649baef825a6788c6
```
Comment (x1ddos) [inline releases/firmware-v9.0.0/assertion.txt:8]: Matches with mine. :+1:

**Comment by benma** [comment]
Cool, after this is merged it would be nice if you added your assertion as well.

**Pull request merged**